src/resolvers/user_management.js

```diff
@@ -30,22 +30,75 @@ export default {
       //   throw new Error('Already logged in.')
       // }
       const session = driver.session()
-      return session.run(
-        'MATCH (user:User {email: $userEmail}) ' +
-        'RETURN user {.id, .slug, .name, .avatar, .email, .password, .role} as user LIMIT 1', {
-          userEmail: email
-        })
-        .then(async (result) => {
+      return session
```

Comment:
```suggestion
      const res = await return session
```
Prefer `async/await`

Reply:
done